src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/NomisData.kt

```diff
@@ -441,6 +445,58 @@ class NomisData(
           builder.apply(dsl)
         }
     }
+
+  // TODO - I chickened out of mapping AGENCY_LOCATIONS because AGY_LOC_ID is everywhere. Left it for the person who migrates agencies.
+  override fun agencyAddress(
```

Comment:
This was a bit naughty - leaving address agency as a top level entity in the test data builder DSL, when it should be a child of agency. However, that looks like a big job unrelated to my needs.